esbuild.mjs

```diff
@@ -13,16 +13,20 @@ if (!existsSync(dist)) {
 const script = await esbuild.build({
   entryPoints: ["src/index.ts"],
   bundle: true,
-  minify: true,
+  minify: false,
```

Comment:
a particular reason not to minify anymore?

Reply:
oops! I'll revert